src/components/app/header.tsx

```diff
@@ -35,6 +37,20 @@ const Header: FunctionComponent = () => {
       ),
     )
 
+  let ActiveCTA: React.FC = () => null
+
+  switch (true) {
```

Comment:
the first case below that is `true` will win.

we might consider changing it to "become a member" as a default for folks that aren't? lots of ways to adjust this.

Reply:
+1 to "Become a Member" as default CTA for now